GPU/Common/TextureDecoder.cpp

```diff
@@ -330,3 +330,339 @@ void DecodeDXT5Block(u32 *dst, const DXT5Block *src, int pitch) {
 		dst += pitch;
 	}
 }
+
+#ifdef _M_SSE
+static inline u32 CombineSSEBits(const __m128i &v) {
```

Comment:
This could be named a bit more descriptively, I think. CombineBitsInSSEWords? or something like that...